### PR TITLE
bump deployment target to iOS 12

### PR DESCRIPTION
**Pull request by mltbnz**
## 📝 Docs

- [x] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?

## 📲 What

Bump minimum deployment target to iOS 12

## 🤔 Why

As agreed upon this bumps the projects deployment target to iOS 12

### CODE REVIEW

File: CriticalMass/NextRidesRequest.swift [hunk 1]
```diff
@@ -7,8 +7,8 @@ struct NextRideQuery: Codable {
     let centerLatitude: CLLocationDegrees
     let centerLongitude: CLLocationDegrees
     let radius: Int
-    let year = Date.getCurrent(\.year)
-    let month = Date.getCurrent(\.month)
+    var year = Date.getCurrent(\.year)
```
Comment (fbernutz) [inline CriticalMass/NextRidesRequest.swift:10]: Was this change intended?

**Review by fbernutz — APPROVED** [review]
Looks good for me. I'm reviewing from my phone so I depend on the green tests 👌😉 Is there maybe an information about the iOS version in the readme or so, which needs to be updated?

File: CriticalMass/NextRidesRequest.swift (hunk 1 above)
Comment (mltbnz) [reply CriticalMass/NextRidesRequest.swift:10]: yes. there was a warning which I wanted to rid of.
Also I think this is the way to go when declaring properties that have a default value on structs

**Comment by mltbnz** [comment]
> Looks good for me. I'm reviewing from my phone so I depend on the green tests 👌😉 Is there maybe an information about the iOS version in the readme or so, which needs to be updated?

I haven't seen additional info in the README but maybe good if I check other docs again
